### ISSUE TITLE
Low accuracy on multi-clsss problem

### ISSUE THREAD

**Issue by IzouGend** [megvii-model/ShuffleNet-Series#50]
When I used shufflenet to train models on my own dataset, I found shufflenet has high accuracy on two-class classification probelm while low accuracy on multi-class problem.
I used finetuning based on officially released pretrained model. Is it unproper to use pretrained model when training multi-class dataset?

**Comment by nmaac** [comment]
@IzouGend It is ok to use the ImageNet pretrained model because ImageNet is a multi-class classification task.
